cfme/configure/access_control.py

```diff
@@ -1405,6 +1407,29 @@ def set_quota(self, **kwargs):
             self.obj_type, self.name))
         assert view.is_displayed
 
+    @property
+    def quota(self):
+        view = navigate_to(self, 'Details')
+        quotas = {
+            'cpu': 'Allocated Virtual CPUs',
+            'memory': 'Allocated Memory in GB',
+            'storage': 'Allocated Storage in GB',
+            'num_vms': 'Allocated Number of Virtual Machines',
+            'templates': 'Allocated Number of Templates'
+        }
+
+        for field in quotas.keys():
+            for item in view.table.rows():
+                if item.name.text == quotas[field]:
+                    quotas[field] = {
+                        'total': item.total_quota.text,
+                        'in_use': item.in_use.text,
+                        'allocated': item.allocated.text,
+                        'available': item.available.text
+                    }
```

Comment:
I think you can replace the for + if with this:
```python
item = view.table.row(name=quotas[field])
```
It should be faster and more obvious.

Reply:
Changed.

Reply:
Why the `for` is still there? The code hints that it should only process one row, why?

Reply:
@mfalesni Corrected.